docs/src/main/asciidoc/_developer-guide/JTA_Reference.adoc

```diff
@@ -494,3 +494,4 @@ or you can list all the configuration options of the subsystem in `jboss-cli`
 ```bash
 /subsystem=transactions:read-resource-description(recursive=true)
 ```
+NOTE: References in this document to CDI refer to the Jakarta Contexts and Dependency Injection unless otherwise noted
```

Comment:
s/to the Jakarta/to Jakarta/g

Also the sentence should end in a period.

Please apply the same to the other uses of this sentence in the PR.

Reply:
Thanks! Done :)